db/schema.rb

```diff
@@ -23,6 +23,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.text "notes"
+    t.uuid "checked_by_id"
```

Comment:
Have we captured somewhere that we'll want to drop the `checked_by` column that this work makes redundant?

Reply:
I've added a checkbox in the card in Trello